packages/eas-cli/src/project/publish.ts

```diff
@@ -140,11 +143,60 @@ export async function buildUpdateInfoGroupAsync(assets: CollectedAssets): Promis
   return updateInfoGroup as UpdateInfoGroup;
 }
 
+export async function buildBundlesAsync({
+  projectDir,
+  inputDir,
+}: {
+  projectDir: string;
+  inputDir: string;
+}) {
+  const packageJSON = await readJSON(path.resolve(projectDir, 'package.json'));
+  if (!packageJSON) {
+    throw new Error('Could not locate package.json');
+  }
+  const isCLIInstalled =
+    !!packageJSON.devDependencies['expo-cli'] || !!packageJSON.dependencies['expo-cli'];
+  if (!isCLIInstalled) {
+    throw new Error(
+      'expo-cli is not installed. If you would like us to handle bundling your app, please install it with: yarn add -D expo-cli'
```

Comment:
```suggestion
      'expo-cli is not installed. To bundle your app, install expo-cli, or skip bundling with --skip-bundle'
```

Reply:
Also note that we currently ask people to install `expo-cli` globally, in the documentation https://docs.expo.io/get-started/installation/#installing-expo-cli

Reply:
It still seems like a good idea to give them a suggestive error message, even if they have ignored our good advice... 😅

Reply:
Sure! The message initially included `yarn add -D expo-cli`, which installs it locally, that's what I meant.